go-controller/pkg/ovn/dns-name-resolver/egressfirewall_dns.go

```diff
@@ -189,7 +190,7 @@ func (e *EgressDNS) addToDNS(dnsName string) {
 //     and the durationTillNextQuery is updated
 //  2. e.added is received and durationTillNextQuery is recomputed
 //  3. e.deleted is received and coincides with dnsName
-func (e *EgressDNS) Run(defaultInterval time.Duration) {
+func (e *EgressDNS) Run(defaultInterval time.Duration) error {
```

Comment:
since `defaultInterval` is only used by this implementation and not the new one, do you think we can move this parameter to the `EgressDNS` constructor and make this method just `Run()`?